packages/pipeline-editor/src/PropertiesPanel/properties-utils.test.ts

```diff
@@ -17,7 +17,7 @@
 import { fillPropertiesWithSavedData } from "./properties-utils";
 
 describe("fillPropertiesWithSavedData", () => {
-  it("should not fill properties that are not defined in schema", () => {
+  it("should fill properties that are not defined in schema", () => {
```

Comment:
what do you mean here? If a value is not defined in the schema, where the values are coming from?

Reply:
there is a `current_parameters` object in common properties, that holds the current value of the property. I thought that it needed to have all keys specified, but it doesn't.

so if you specify `current_parameters` as:
```
current_parameters: {
  thing1: "one",
  thing2: "two",
}
```

but you try to set `thing3`, it won't persist the value, because I considered it as an invalid key, but that is not necessarily the case

Reply:
Could change the test name to:
```
"should fill properties that are not explicitly defined in the defaults"
```